api/services/Gus/connection.js

```diff
@@ -0,0 +1,43 @@
+const jsforce = require('jsforce');
+const logger = require('../../services/Logs/logger');
+
+let connection;
+async function getConnection() {
+    if (connection) {
+        return connection;
+    }
+
+    const conn = new jsforce.Connection({ logLevel: 'DEBUG' });
+    try {
+        await conn.login(process.env.GUS_USERNAME, process.env.GUS_PASSWORD);
+        connection = conn;
+
+        // Keep connection open, but do reconnect every so often
+        setTimeout(() => {
+            connection = null;
+            logger.info(`Forgetting Gus session`);
+        }, 10 * 60 * 1000); // 10 minutes
```

Comment:
This may not be necessary since jsforce already auto-refreshes its connection from what I see at https://github.com/jsforce/jsforce/blob/master/lib/connection.js#L1461-L1479

Were you experiencing issues?

Reply:
Interesting! I was actually trying to reuse connections here, since the old code created logged in for every request.
No issues seen actually -- I'll remove this block so the auto-refresh can be used.